examples/annotation_import/image.ipynb

```diff
@@ -8,7 +8,7 @@
       },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    mal_label_list = [li_label]
</pre><p>shouldn't this be called <code>li_label_list</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/744/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
<p>Yes, thank you for catching that !</p>